upgrade-plugins/src/main/java/org/exoplatform/ecms/upgrade/templates/NodeTypeTemplateUpgradePlugin.java

```diff
@@ -67,6 +76,24 @@ public void processUpgrade(String oldVersion, String newVersion) {
       log.info("Start " + this.getClass().getName() + ".............");
     }
     String unchangedNodeTypes = PrivilegedSystemHelper.getProperty("unchanged-nodetype-templates");
+    String previousPlfVersion = PRODUCT_VERSION_ZERO;
+    Set<String> modifiedTemplateLog = new HashSet<String>();
+    try {
+      modifiedTemplateLog = Utils.getAllEditedConfiguredData(TemplateServiceImpl.class.getSimpleName(),
+                                                                         EDITED_CONFIGURED_NODE_TYPES,true);
+    } catch (Exception e2) {
+      if (log.isInfoEnabled()) {
+        log.info("Can not get All Edited Template log");
```

Comment:
An exception is thrown, it should be a log.error here (and no need for the test before).
Also, do not swallow the exception.
This comment is true for others similar cases in the PR.